apps/server/lib/lexical/server/code_intelligence/completion/env.ex

```diff
@@ -475,4 +512,16 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Env do
         {:ok, tokens, ''}
     end
   end
+
+  defp path_to_cursor(env) do
```

Comment:
this function is called `path_to_cursor` but it takes an `env` and returns a path`

Reply:
I think `cursor_path/1` would be better.